Package.swift

```diff
@@ -738,6 +738,16 @@ let package = Package(
     swiftLanguageVersions: [.v5]
 )
 
+#if canImport(Darwin)
+package.targets.append(contentsOf: [
+    .executableTarget(
+      name: "swiftpm-testing-helper",
+      dependencies: ["Commands"],
+      exclude: ["CMakeLists.txt"]
```

Comment:
We still need to teach CMake about this executable, right?

Reply:
I had that originally but it's not part of the regular build and bootstrap is using freshly build swiftpm so I think we are good.